mappings/net/minecraft/client/gui/screen/world/EditGameRulesScreen.mapping

```diff
@@ -50,3 +50,13 @@ CLASS net/minecraft/class_5235 net/minecraft/client/gui/screen/world/EditGameRul
 			METHOD method_27640 createRuleWidget (Lnet/minecraft/class_1928$class_4313;Lnet/minecraft/class_5235$class_5238;)V
 				ARG 1 key
 				ARG 2 widgetFactory
+	CLASS class_5400 NamedRuleWidget
+		FIELD field_25629 name Ljava/util/List;
+		FIELD field_25630 children Ljava/util/List;
+		METHOD <init> (Lnet/minecraft/class_5235;Ljava/util/List;Lnet/minecraft/class_2561;)V
+			ARG 2 description
+			ARG 3 name
+		METHOD method_29989 render (Lnet/minecraft/class_4587;II)V
```

Comment:
This renders the name of the rule on the left side of the screen. Not actually a `render` method. The widgets are rendered by implementations of the NamedRuleWidget

Reply:
`drawName` or `renderName`?